jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile

```diff
@@ -116,12 +108,11 @@ String findLatestVersionFor(String product, String currentVersion, List allVersi
             latestVersion = version
             return true
         } else {
-            boolean isGA = verifyIsGA(product, version)
-            if (type == 'ga' && isGA) {
+            if (type == 'ga') {
                 latestVersion = version
                 return true
             }
-            if (type == 'rc' && !isGA) {
+            if (type == 'rc') {
                 latestVersion = version
                 return true
```

Comment:
I know this seems pointless, but could we just leave the verifyIsGa logic in place for the moment and just have it always return true.

We will likley have to come back to this and would prefer to leave it here so i dont forget where i needed to deal with this.

Also, if you were to remove this, you could replace this entire thing https://github.com/integr8ly/ci-cd/blob/d7cce0d16e929bb6dcdf6dfebd9d44b9a9105ebf/jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile#L105-L120 with:

```
String latestVersion = matchingVersions.first()
```

Reply:
Yeah sure. I'll fix that up now. 👍